caffe2/python/pybind_state.cc

```diff
@@ -1502,6 +1503,27 @@ void addGlobalMethods(py::module& m) {
     new_proto.SerializeToString(&out);
     return py::bytes(out);
   });
+  m.def(
+      "transform_onnxifi",
+      [](const py::bytes& pred_net_str,
+         const std::unordered_map<std::string, std::vector<int>>& shapes,
+         bool debug_builder) -> py::bytes {
+        caffe2::NetDef pred_net;
+        if (!ParseProtoFromLargeString(
+                pred_net_str.cast<std::string>(), &pred_net)) {
+          LOG(ERROR) << "broken pred_net protobuf";
```

Comment:
After logging you fall-through to execute subsequent code, should raise Python exception